components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceComponent.java

```diff
@@ -901,6 +922,10 @@ protected void setMultiAttributeLoginService(MultiAttributeLoginService multiAtt
         FrameworkServiceDataHolder.getInstance().setMultiAttributeLoginService(multiAttributeLogin);
     }
 
+    /**
+     * @deprecated To generalize the resolver concept and make it extensible.
+     * Use the {@link #unsetLoginResolverService(LoginResolverService)} method instead.
+     */
```

Comment:
If this service is not going to be used we can remove the reference as well. This may improve the server start-up. wdyt?

Reply:
Yes, you are correct. However, it was marked as deprecated and kept as it is to maintain backward compatibility hence we can announce the deprecation notice in the 6.1.0 release and then completely remove the implementation/services/etc.. in the 6.2.0 release.

Reply:
not sure if we need to do it as this is an internal change of the architecture. We only needs to mention the behavioral change from UI level IMO